### PR TITLE
Referencing issue + fix for React/Radium warnings

### PR DESCRIPTION
**Pull request by JackuB**
Extracted fix from https://github.com/apiaryio/attributes-kit/pull/447

### CODE REVIEW

**Comment by JackuB** [comment]
[Attributes Kit is failing for references to another data structure](https://trello.com/c/Ic4LVC9r/292-attributes-kit-is-failing-for-references-to-another-data-structure)

File: src/Modules/ElementUtils/inheritance.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ const INHERITED_MEMBER_QUERY = {
   $or: [
     {
       'meta.links': {
-        href: 'http://example.com/inherited-member/',
+        href: 'http://refract.link/inherited-member/',
```
Comment (michalholasek) [inline src/Modules/ElementUtils/inheritance.js]: @JackuB Could you please elaborate why is this change needed to fix referencing issue?

Comment (JackuB) [inline src/Modules/ElementUtils/inheritance.js]: It's in the commit message - https://github.com/apiaryio/attributes-kit/pull/449/commits/6f8d36882304cbe5987df9e51c59c148a9dd8aae

Eidolon relies on this string

Comment (michalholasek) [inline src/Modules/ElementUtils/inheritance.js]: Ah, I see.

**Comment by JackuB** [comment]
@michalholasek ready for review, had to rebuild without cache to fix tests

File: test/fixturesComparision.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ describe('Comparision with reference fixtures', () => {
         );
 
         it('They should be equal', () => {
-          assert.equal(htmlString, reference);
+          assert.equal(htmlString.replace(/\n */g, ''), reference.replace(/\n */g, ''));
```
Comment (michalholasek) [inline test/fixturesComparision.js]: Could you please elaborate a little bit on this?

**Comment by michalholasek** [comment]
@JackuB You added other fixes from #447, hence this PR is not just about "referencing issue" - it might be appropriate to change the title.

File: test/fixturesComparision.js (hunk 2 above)
Comment (JackuB) [inline test/fixturesComparision.js]: My bad, shouldn't be there. Simplified commit history of this PR

**Comment by JackuB** [comment]
@michalholasek updated title
